tensorflow/python/kernel_tests/nn_ops/rnn_cell_test.py

```diff
@@ -1354,6 +1354,35 @@ def testLSTMBlockCellErrorHandling(self):
               forget_bias=forget_bias,
               cell_clip=cell_clip,
               use_peephole=use_peephole))
+      
+  @test_util.run_gpu_only
```

Comment:
We should have the same behavior between CPU and GPU - so the test should be common.  There may be differences in error messages, but this is usually handled by using a regex that accounts for both down below.